### ISSUE TITLE
Decoration views not being offset by supplementary views

### ISSUE THREAD

**Issue by notjosh** [kishikawakatsumi/IBPCollectionViewCompositionalLayout#77]
This might be a little hard to follow, but refer to the image below.

A section is configured with a `NSCollectionLayoutDecorationItem.background(elementKind:)` (the grey background), and it also has a `boundarySupplementaryItem` (the header that says "Kategorien"). No content insets, nothing fancy.

The cell itself ("Information zu Krebs") matches the height of the decoration item (✅), but the y-position is aligned with the header, not the cell.

<img width="461" alt="image" src="https://user-images.githubusercontent.com/33126/63557658-7975a080-c4fe-11e9-81c3-ba788eaa4470.png">

This regression was introduced a couple of days ago when adding support for pinned headers, I think.

I'll try to track it down in the meantime, but you might know the solution more quickly :)

**Comment by kishikawakatsumi** [comment]
@notjosh Could provide a reproducible code? If possible, it would be a great help to investigate the issue. Thank you for reporting!

**Comment by notjosh** [comment]
Yeah, sure. I'm actually not sure the correct behaviour here actually - maybe it shouldn't be offset, but the height should include the header too? I'll create an example in my fork soon!

**Comment by kishikawakatsumi** [comment]
If it runs on iOS 13 and the behavior is different, it is almost the library's bug. 😅

**Comment by notjosh** [comment]
Ok, repro up on branch: https://github.com/notjosh/IBPCollectionViewCompositionalLayout/tree/sample-supplementary-with-decoration

iOS 12.4:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/33126/63560995-1dfedf00-c50d-11e9-801f-d5e899395fb2.png">

iOS 13:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/33126/63560996-222afc80-c50d-11e9-9d98-dacdcc450d74.png">
